src/components/Tables/RequestTable/RequestTable.tsx

```diff
@@ -336,7 +336,7 @@ const RequestTable: React.FC<RequestTableProps> = ({
 				),
 			},
 			{
-				Header: () => <div className={styles.left}>Artwork Info</div>,
+				Header: () => <div className={styles.left}>DOMAIN</div>,
```

Comment:
Make it "Domain" to be  consistent with other headers -- they're all capitalised in CSS. Very minor nitpick, wouldn't have put it in if there were no other changes but may as well change it over since it's a very quick one

Reply:
Done.